arch/arm/core/aarch32/cortex_m/mpu/arm_core_mpu.c

```diff
@@ -82,6 +87,14 @@ static const struct z_arm_mpu_partition static_regions[] = {
 		.attr = K_MEM_PARTITION_P_RX_U_RX,
 		},
 #endif /* CONFIG_ARCH_HAS_RAMFUNC_SUPPORT */
+#if defined(CONFIG_CODE_DATA_RELOCATION_SRAM)
+		{
+		/* RAM area for relocated text */
+		.start = (uint32_t)&__sram_text_start,
+		.size = (uint32_t)&__sram_text_size,
+		.attr = K_MEM_PARTITION_P_RX_U_RX,
```

Comment:
This partition has exactly the same attribution as the ramfunc section. I wonder if that one could be re-used. Not against, for now, to accept adding one more region. But i really do not see why we need to have 2 static regions with the same attribution.